packages/idempotency/src/types/IdempotencyOptions.ts

```diff
@@ -1,9 +1,11 @@
 import type { Context } from 'aws-lambda';
 import { BasePersistenceLayer } from '../persistence/BasePersistenceLayer';
+import { IdempotencyConfig } from 'IdempotencyConfig';
 
-type IdempotencyOptions = {
-  dataKeywordArgument: string
+type IdempotentOptions = {
   persistenceStore: BasePersistenceLayer
+  dataKeywordArgument?: string
```

Comment:
Can we please add docstrings for each property like below? This way when someone hovers it it's clear what they are.